commands/version/command.js

```diff
@@ -140,6 +140,10 @@ exports.builder = (yargs, composed) => {
       alias: "yes",
       type: "boolean",
     },
+    "changelog-preview": {
```

Comment:
This should be alphabetized after the `changelog-preset` key (yes, _behind_ conventional-changelog, ordering is... challenging)